MH4U Dex/MonsterItemTableViewController.h

```diff
@@ -8,9 +8,10 @@
 
 #import <UIKit/UIKit.h>
 
-@interface MonsterLowRankTableViewController : UITableViewController
+@interface MonsterItemTableViewController : UITableViewController
 
 @property (nonatomic, strong) NSManagedObjectContext *managedObjectContext;
 @property (nonatomic, strong) NSString *monster;
```

Comment:
I wouldn't expect a property named `monster` to be a string—should this be a `Monster` object or should it be called `monsterName` or something else?